src/pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -30,7 +30,7 @@
     from torch.cuda.amp import autocast as old_autocast
 
 
-class NativeMixedPrecisionPlugin(MixedPrecisionPlugin):
+class NativeMixedPrecisionPlugin(PrecisionPlugin):
```

Comment:
Is there a plan to merge these with the lite plugins, build ontop of the lite plugins?

Reply:
Right now they can't because the base class in PL has PL-specific methods (e.g. connect). I don't see that changing very soon. The same also in strategies.